### PR TITLE
[Panorama] Tags Management

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6356)

## Description
CRUD commands for tags management in Panorama.

## Screenshot
Screenshot 2023-06-01 at 12 00 12

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -13069,6 +13069,298 @@ def pan_os_delete_application_group_command(args):
...
+    Args:
+        args (dict): The command arguments to edit the tag.
+
+    Returns:
+        CommandResults: The command results with raw response and readable outputs.
+    """
+    tag_name = args.get('name', '')
+    new_tag_name = args.get('new_name', tag_name)
+    disable_override = argToBoolean(args.get('disable_override', False))
+    comment = args.get('comment', '')
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:13306]: what about `device_group` arg? is it needed?

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 2]
```diff
@@ -13069,6 +13069,298 @@ def pan_os_delete_application_group_command(args):
...
+def pan_os_list_tag_command(args: dict) -> CommandResults:
+    """Sends the request and returns the result of the command pan-os-list-tag.
+
+    Args:
+        args (dict): The command arguments.
+
+    Returns:
+        CommandResults: The command results with raw response, outputs and readable outputs.
+    """
+    include_shared = argToBoolean(args.get('include_shared_tags', False))
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:13177]: same

**Review by dantavori — COMMENTED** [review]
great!
you can schedule a demo for next week.
what about this point in the DOD?
```
Add description to the integration that refer to SASE with an explanation of when each should be used.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (yaakovpraisler) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:13306]: The device-group argument is handled ([here](https://github.com/demisto/content/blob/panorama-tags-management/Packs/PAN-OS/Integrations/Panorama/Panorama.py#L8132)) in the `initialize_instance` method which is called in `main`
I do see now that it is different called for args (with `-` instead `_`) I'll push a fix.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 3]
```diff
@@ -213,7 +213,7 @@ configuration:
   required: false
   type: 13
   section: Connect
-description: Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.
+description: Manage Palo Alto Networks Firewall and Panorama. Use this pack to manage Prisma Access through Panorama. For more information see Panorama documentation
```
Comment (dantavori) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:216]:
```suggestion
description: Manage Palo Alto Networks Firewall and Panorama. Use this pack to manage Prisma Access through Panorama. For more information see Panorama documentation.
```
does this change meet the DoD of "adding a description to the integration that refers to SASE with an explanation of when each should be used"?

Comment (yaakovpraisler) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:216]: Yes

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 4]
```diff
@@ -8673,3 +8673,147 @@ There is no context output for this command.
...
+```!pan-os-edit-tag name="testtag" new_name="newtesttag" comment="some comment"```
+
+#### Human Readable Output
+
+>The tag with name "testtag" was edited successfully.
+
+### pan-os-delete-tag
+
+***
+Deletes a tag from panorama.
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/Integrations/Panorama/README.md:8798]:
```suggestion
Deletes a tag from Panorama.
```

File: Packs/PAN-OS/ReleaseNotes/1_17_5.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Added the following new commands to manage Panorama tags:
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/1_17_5.md:6]:
```suggestion
Added the following new commands to manage Panorama tags:
```

File: Packs/PAN-OS/ReleaseNotes/1_17_5.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  - ***pan-os-list-tag***
+  - ***pan-os-create-tag***
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/1_17_5.md:8]:
```suggestion
- ***pan-os-create-tag***
```

File: Packs/PAN-OS/ReleaseNotes/1_17_5.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+  - ***pan-os-edit-tag***
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/1_17_5.md:9]:
```suggestion
- ***pan-os-edit-tag***
```

File: Packs/PAN-OS/ReleaseNotes/1_17_5.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+  - ***pan-os-delete-tag***
```
Comment (ShirleyDenkberg) [inline Packs/PAN-OS/ReleaseNotes/1_17_5.md:10]:
```suggestion
- ***pan-os-delete-tag***
```

**Review by dantavori — APPROVED** [review]
great! see one note below

**Pull request merged**